deepstream/deepstream-dev-pod.Dockerfile

```diff
@@ -36,7 +40,12 @@ RUN apt-get install -y --no-install-recommends \
         python3-dev \
         python3-gi \
         python3-gst-1.0 \
-        python3-pip
+        python3-pip \
```

Comment:
These dependencies are all good. However, please keep the lists in alphabetical order to improve maintainability.

Reply:
done